### ISSUE TITLE
Exiting executable during input prompts results in traceback

### ISSUE THREAD

**Issue by jaredhendrickson13** [jaredhendrickson13/pfsense-automator#4]
Running from Windows 10, user has experienced traceback when sigint (ctl+c) is passed during input. This does not disrupt functionality but is an unintended annoyance.

```
C:\Users\jhendrickson\pfsense-automator_v0.0.2_windows10>pfsense-automator.exe test.jh.co --check-auth -u admin
```                                                                                                                       
```
WARNING: Large numbers of authentication failures will enforce a pfSense lockout for your IP address. Proceed with caution.
Please enter password: 
Traceback (most recent call last):
File "pfsense-automator.py", line 1016, in <module>
File "pfsense-automator.py", line 945, in main
File "Users\jhendrickson\AppData\Local\Programs\Python\Python37\lib\getpass.py", line 110, in win_getpass
KeyboardInterrupt
[10708] Failed to execute script pfsense-automator
```

**Comment by jaredhendrickson13** [comment]
This also seems to only effect password inputs

**Comment by jaredhendrickson13** [comment]
Issue has been fixed in commit `b92993645877c6f507892dd0330b09c839a87a4c` will be available in v0.0.4
